setup.cfg

```diff
@@ -35,6 +35,7 @@ per-file-ignores =
     linkcheck/__init__.py: E402,F401
     linkcheck/bookmarks/chrome.py: E402
     linkcheck/checker/httpurl.py: E402
+    linkcheck/configuration/__init__.py: F401
```

Comment:
I'm guessing this is from `import gi` and it would go away if you called `gi.require_version()`.

Reply:
Done.